keyboards/quokka/quokka.c

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2023 @dlford
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "quokka.h"
+
+// clang-format off
+
+#ifdef RGB_MATRIX_ENABLE
+
+// LED Layout
+// Columns
+// 0   1   2   3   4   5   6   7   8   9   10  11
+// Physical (Center: 133)
+// 20  41  61  81  102 122 143 163 183 204 224 244
+//                                                  Rows  Physical (Center: 54)
+// 17  12  11  06  05          23  24  29  30  35   0     21
+// 16  13  10  07  04          22  25  28  31  34   1     43
+// 15  14  09  08  03          21  26  27  32  33   2     64
+//             02  01  00  18  19  20               3     85
+
+led_config_t g_led_config = { {
```

Comment:
This can and should be moved to the info.json file.  You can use `qmk info -kb quokka -f json` to help generate a lot of the config

Reply:
Thanks for the tip! This has been addressed.